docs/src/components/ButtonLink.tsx

```diff
@@ -163,6 +163,7 @@ export default function ButtonLink<T extends React.ElementType = typeof DefaultC
   const Component = asProp || DefaultComponent;
 
   return (
+    // @ts-expect-error react-polymorphic-types is currently not compatible with styled-components
```

Comment:
Is this in the scope of the PR? Just curious why it was necessary to use suppress comment right now.

Reply:
because it caused error in the pipeline 🤔

Reply:
👌